Add sub via protocol mapper to access token (#28540)
keycloak/keycloak: docs/documentation/server_admin/topics/clients/con-protocol-mappers.adoc

When scripts deploy, you should be able to select the deployed scripts from the list of available mappers.

== Pairwise subject identifier mapper

Subject claim _sub_ is mapped by default by *Subject (sub)* protocol mapper in the default client scope *basic*.

> In case you want to use a pairwise subject identifier via protocol mapper like *Pairwise subject identifier* you should remove the *Subject (sub)* protocol mapper from the *basic* client scope.

Reviewer: ```suggestion
To use a pairwise subject identifier by using a protocol mapper such as *Pairwise subject identifier*, remove the *Subject (sub)* protocol mapper from the *basic* client scope.
```

Information that cannot be removed from a lightweight access token::
  Protocol mappers can controls which information is put onto an access token and the lightweight access token use the protocol mappers. Therefore, the following information cannot be removed from the lightweight access. +

>   `exp`, `iat`, `jti`, `iss`, `typ`, `azp`, `nonce`, `sid`, `scope`, `cnf`

Reviewer 2: Nitpick: It can be good to remove also `nonce` from this list. It was removed from lightweight access token in some of previous PRs.

However this is not a blocker for this PR as we can do in the follow-up documentation issue we have. Just in case that @andymunro found some other things during docs review.

Author: Thanks, removed `nonce` claim.
